apis/Google.Cloud.Iam.V1/Google.Cloud.Iam.V1/PolicyPartial.cs

```diff
@@ -77,5 +89,8 @@ public bool RemoveRoleMember(string role, string member)
         }
 
         private Binding FindRole(string role) => Bindings.FirstOrDefault(binding => binding.Role == role);
+
+        private void ValidatePolicyVersion() =>
+            GaxPreconditions.CheckState(Version <= 1, "Helper methods cannot be invoked on policies with version {0}. For more information, see FIXME.", Version);
```

Comment:
Note: we still need to resolve this FIXME, which requires some documentation to exist somewhere